pkg/webhook/server.go

```diff
@@ -180,6 +182,13 @@ func (s *Server) mutate(receivedAdmissionReview v1beta1.AdmissionReview) *v1beta
 		meshName = v
 	}
 
+	// set egress ignored ports
+	if v, ok := pod.ObjectMeta.Annotations[egressIgnoredPortsAnnotation]; ok {
+		egressIgnoredPorts = v
+	} else {
+		egressIgnoredPorts = "22"
```

Comment:
I don’t think port 22 should be excluded by default. The egress ignored ports env var should be set only if the annotation exists.

Reply:
That's what the mesh does by default - https://docs.aws.amazon.com/app-mesh/latest/userguide/mesh-getting-started-ec2.html

I'm just following entrypoint script aws uses - ```# Comma separated list of ports for which egress traffic will be ignored, we always refuse to route SSH traffic.
if [ -z "$APPMESH_EGRESS_IGNORED_PORTS" ]; then
    APPMESH_EGRESS_IGNORED_PORTS="22"
else
    APPMESH_EGRESS_IGNORED_PORTS="$APPMESH_EGRESS_IGNORED_PORTS,22"
fi```